bower.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "adapt-contrib-resources",
-  "version": "5.0.1",
+  "version": "5.0.2",
```

Comment:
version numbers are always bumped by releaser, please revert in the readme and bower.json